### DISCUSSION TITLE
Will copilot be upgrading to use the GPT 3.5 like chat gpt.

### DISCUSSION THREAD

**Discussion by tmlewin** [community/community#41387] [Copilot Conversations]
### Select Topic Area

Question

### Body

Chat gpt is a step above everything i have seen . Would copilot be upgrading to 3.5

**Comment by ilyasdotdev** [comment]
I use github copilot, It will be very nice if github uses gpt3.5

**Comment by MulleDK19** [comment]
CoPilot uses a 12 billion parameter model, ChatGPT uses a 175 billion parameter model. Unless you wanna pay $150 a month and wait 10 times longer for suggestions, that's not going to happen any time soon..

**Reply by kasthack** [reply]
Actually, I wouldn't mind having that option. The performance boost from having an AI generate much better code is worth it.

**Comment by rampatra** [comment]
Any updates? Wondering why no answers on this yet. Quite curious to know.

**Comment by fara1234589** [comment]
GitHub Copilot, GitHub's AI-driven programmer's assistant, is set to add chat and voice capabilities, as the tool adopts OpenAI's ChatGPT-4 deep learning technology.
Regards: [cpsandtypingtest](https://cpsandtypingtest.com/)
